data/accountManager.go

```diff
@@ -174,22 +170,6 @@ func (manager *AccountManager) DeleteOldKeys(latestHdr bookkeeping.BlockHeader,
 		// We need a key for round r+1 for agreement.
 		nextRound := latestHdr.Round + 1
 
-		if latestHdr.CompactCert[protocol.CompactCertBasic].CompactCertNextRound > 0 {
```

Comment:
why did this code go?

Reply:
We wont need this due to the falcon keys which uses different set of keys.